ckan/tests/logic/action/test_patch.py

```diff
@@ -214,6 +214,31 @@ def test_user_patch_updating_single_field(self):
         assert user2["fullname"] == "Mr. Test User"
         assert user2["about"] == "somethingnew"
 
+    def test_extensions_successfull_patch_updating_user_name(self):
+        user = factories.User()
+
+        updated_user = helpers.call_action(
+            "user_patch",
+            context={"user": None, "ignore_auth": True},
+            **{'id': user["id"], 'name': "somethingnew"}
+        )
+
+        assert updated_user["name"] == "somethingnew"
+
+        user2 = helpers.call_action("user_show", id=user["id"])
+
+        assert user2["name"] == "somethingnew"
+
+    def test_extensions_failed_patch_updating_user_name(self):
+        user = factories.User()
+
+        with pytest.raises(ValidationError):
+            helpers.call_action(
+            "user_patch",
+            context={"user": None, "ignore_auth": True},
```

Comment:
Here we want to test the standard scenario, i.e. that user_patch will fail when trying to update the name (if ignore_auth is missing or False)

```suggestion
            context={"user": None, "ignore_auth": False},
```